Description: Avoid generating invalid frame number 75
 The conditional increment to the frame number, which is probably meant to
 avoid loosing the last incomplete frame in the very last track of a disk, has
 the side effect of generating frame numbers equal to 75, when they should be
 between 0 and 74 included, which results in an unburnable image.
 .
 While the code could be adapted to count the last frame, it is clearer and
 less intrusive to the original code to simply remove the conditional increment,
 and possibily loosing 1/75th a second on the very last track should not matter
 to anyone.
Author: Tanguy Ortolo <tanguy+debian@ortolo.eu>
Last-Update: 2011-11-10
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=619723
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=664756

Index: brasero-3.0.0/plugins/audio2cue/burn-audio2cue.c
===================================================================
--- brasero-3.0.0.orig/plugins/audio2cue/burn-audio2cue.c	2010-08-16 04:55:07.000000000 +0200
+++ brasero-3.0.0/plugins/audio2cue/burn-audio2cue.c	2011-11-10 23:44:10.232078473 +0100
@@ -317,7 +317,7 @@
 	else
 		frame = len * 75;
 
-	frame = frame / 1000000000 + ((frame % 1000000000LL) ? 1:0);
+	frame = frame / 1000000000;
 
 	len /= 1000000000LL;
 	min = len / 60;
